winnativeeventfilter.cpp

```diff
@@ -1309,6 +1310,8 @@ bool WinNativeEventFilter::nativeEventFilter(const QByteArray &eventType,
                     QPointF point = {obj->property("x").toReal(), obj->property("y").toReal()};
                     for (QObject *parent = obj->parent(); parent; parent = parent->parent()) {
                         point += {parent->property("x").toReal(), parent->property("y").toReal()};
+                        if(parent->isWindowType())
```

Comment:
I'd prefer:
```cpp
if (condition) {
    doSomething();
}
```

Reply:
Fixed in c287322.

Reply:
OK, but can you add a space between `if` and `(`? like: `if (`.